src/libs/ReportUtils.ts

```diff
@@ -7339,6 +7339,15 @@ function reasonForReportToBeInOptionList({
         return null;
     }
 
+    const currentReportActions = allReportActions?.[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report?.reportID}`] ?? {};
+    const reportActionValues = Object.values(currentReportActions);
+    const hasOnlyCreatedAction = reportActionValues.length === 1 && reportActionValues.at(0)?.actionName === CONST.REPORT.ACTIONS.TYPE.CREATED;
+
+    // Hide reports that have only a `CREATED` action, a total of 0, and are in a submitted state
```

Comment:
Can you please also update this comment to explain _WHY_ they should be hidden? I think it's primarily because:
1. The report shows as empty
2. There is nothing actionable by the user

Reply:
Good point, I've updated the comment.